extensions/paste-py/src/paste-py.tsx

```diff
@@ -0,0 +1,139 @@
+import { Form, ActionPanel, Action, showToast, Toast, open } from "@raycast/api";
+import { useState, useEffect } from "react";
+
+interface FormValues {
+  content: string;
+  extension: string;
+}
+
+interface Language {
+  name: string;
+  type: string;
+  extensions: string[];
+}
+
+export default function Command() {
+  const [isLoading, setIsLoading] = useState(false);
+  const [extensionOptions, setExtensionOptions] = useState<string[]>([]);
+  const [content, setContent] = useState("");
+  const [extension, setExtension] = useState("txt");
+
+  useEffect(() => {
+    async function fetchLanguages() {
+      try {
+        const response = await fetch("https://paste.fosscu.org/languages.json");
```

Comment:
Worth using the util hooks here for built-in caching and whaatnot? E.g.:

https://developers.raycast.com/utilities/react-hooks/usefetch